en/simulation/flightgear.md

```diff
@@ -124,7 +124,7 @@ INFO  [simulator] Waiting for simulator to accept connection on TCP port 4560
 INFO  [simulator] Simulator connected on TCP port 4560.
 INFO  [commander] LED: open /dev/led0 failed (22)
 INFO  [commander] Mission #3 loaded, 9 WPs, curr: 8
-INFO  [init] Mixer: etc/mixers-sitl/plane_sitl.main.mix on /dev/pwm_output0
+INFO  [init] Mixer: etc/mixers-sitl/plane_sitl.main.mix on /dev/pwm_output0 
```

Comment:
I don't want to run this simulator again just to test this. My thinking is that if I delete this line, very little would be lost right?

Reply:
Yes you can just remove it.